### ISSUE TITLE
Hassgalaxy companion app gives error: No connection for "token"

### ISSUE THREAD

**Issue by pfeldman62** [Jerrkawz/hassgalaxy#2]

**Comment by pfeldman62** [comment]
Hi, I just installed Home Assistant UI app on my Galaxy S2 watch. It created a URL for me with the token, and when I went to that URL in my browser, I got the error: No connection for XXXX (where XXXX is a generated token).
I uninstalled the app from my watch a couple of times. Each time, when I started the app, a URL with the new token was generated and I always got the same error when I tried accessing the URL. Is Hass Galaxy website working OK?
![Capture](https://user-images.githubusercontent.com/43377403/85975587-7e20a600-b9a6-11ea-8261-50d9c1b77714.JPG)

**Comment by mikaelklingstam** [comment]
> Hi, I just installed Home Assistant UI app on my Galaxy S2 watch. It created a URL for me with the token, and when I went to that URL in my browser, I got the error: No connection for XXXX (where XXXX is a generated token).
> I uninstalled the app from my watch a couple of times. Each time, when I started the app, a URL with the new token was generated and I always got the same error when I tried accessing the URL. Is Hass Galaxy website working OK?
> ![Capture](https://user-images.githubusercontent.com/43377403/85975587-7e20a600-b9a6-11ea-8261-50d9c1b77714.JPG)

I have the same issue. Is there another way to solve it pls?
I refuse to type approx 300 letters in the token code field :)

**Comment by six-shooter** [comment]
I am experiencing the same issue, now nearly a year later. did you get it to work?

**Comment by bogdanr** [comment]
Same problem for me.

**Comment by pfeldman62** [comment]
I don't believe it's still supported any longer. I gave up. Looking for some alternative.
